CMakeLists.txt

```diff
@@ -155,12 +161,14 @@ endif ()
 
 set(Boost_USE_MULTITHREADED ON)
 set(Boost_USE_STATIC_RUNTIME OFF)
+
 find_package(Boost 1.81 REQUIRED COMPONENTS json url coroutine)
 message(STATUS "LaunchDarkly: using Boost v${Boost_VERSION}")
 
-include(${CMAKE_FILES}/certify.cmake)
+set(FOXY_BUILD_TESTING OFF)
 add_subdirectory(vendor/foxy)
```

Comment:
I added a new `FOXY_BUILD_TESTING` to avoid the whole "get the previous value of `BUILD_TESTING`, set it to off, `add_subdir`, set the value back.." song and dance.